developers/buildsystem.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: developersguide
+title: Buildsystem
+---
+
+{% include base.html %}
+
+# Build system
+
+The buildsystem is based on maven.
+A very common tool for Java development.
+Maven is a convention centric, declarative system that is extensible via addional plugins.
+That means if you stick 100% to mavens idea of a java project, your buildsystem instruction file is not longer than 10 lines.
+
+openHAB has a few extra requirements and we use about 10 additional plugins,
+ranging from OSGi specific ones (bnd) to publish and testing plugins.
+
+This section talks about some common buildsystem related topics and also some quirks that you will encounter.
+
+## Adding dependencies
+
+Generally all dependencies should be OSGi-bundles and available on JCenter.
+
+*** External dependency ***
```

Comment:
Is this supposed to be some markdown? If so, it does not work...

Reply:
@J-N-K can you help out here and also quickly go over the file if I have forgotten to apply any recent changes? Thanks

Reply:
```suggestion
### External dependency 
```

should do the trick